### DISCUSSION TITLE
Filter subreddit flair combo

### DISCUSSION THREAD

**Discussion by EnArvy** [Docile-Alligator/Infinity-For-Reddit#1717] [Ideas]
### Checklist

- [X] I have used the search function for [open](https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/categories/ideas) **and** [closed](https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/categories/completed-ideas) ideas to see if someone else has already submitted the same request.
- [X] I will describe the feature with as much detail as possible.

### Description

Can we have filter type that takes input a combo of subreddit and flair and filters that flair subreddit combo specifically?
Foe example, input is pics.politics and when applied to Home, any post with flair politics from subreddit pics is filtered out, but not any post from other subs with flair politics?

### Additional context

This is helpful because usually people browse their Home/Multireddits and it is difficult to filter out unwanted flairs from specific subs.

### How important is this feature?

Important

### Solution

I tried coding this up, but am stuck with database migration.

**Comment by imjuzcy** [comment]
I think if the conditions in a post filter's settings could be set to "and"  instead of just "or", this could be easily done. 

For example, currently if in a filter you set to a keyword blacklist "Disney" and subreddit blacklist "AskReddit", all posts from r/AskReddit **and** posts that contain the word "Disney" are hidden. 

This feels like a bug, since you can replicate this with two filters, one with title keyword blacklist "Disney" and subreddit blacklist "AskReddit" and apply the two filters to the same feed.
